### ISSUE TITLE
"Error: Cannof find module 'egrep'" but it is installed

### ISSUE THREAD

**Issue by exoego** [apexearth/egrep#1]
Hello!
I am looking for grep-like node module that can return not only matched file names but also matched lines.

`egrep` seems fit my use case!

Unfortunately, `Error: Cannot find module 'egrep'` occured when I tried to import `egrep`.
Any ideas?

# How to Reproduce

1. Create `package.json`
```json
{
  "scripts": {
    "my": "node myscript.js"
  },
  "devDependencies": {
    "@apexearth/egrep": "1.0.4"
  },
  "dependencies": {  }
}
```

2. Run `npm install`
3. Create `myscript.js` (followed https://github.com/apexearth/egrep#nodejs-usage)
```js
const egrep = require('egrep');
const stream = egrep ({
  pattern: /foo/,
  files: [ '**/*.csv'  ],
  recursive: true
});
stream.on('data', data => {
  console.log(data)
});
stream.on('error', err => {
  console.log(err)
});
stream.on('close', () => {
  console.log('closed')
});
```

4. Run `npm run my`

```bash
$ npm run my

> @ my /home/vagrant/IdeaProjects/experiment
> node myscript.js

module.js:549
    throw err;

Error: Cannot find module 'egrep'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/vagrant/IdeaProjects/annofab/script/check-export-name.js:1:77)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

Look like installation is done successfully.

There is an entry in `package-lock.json` like below and `node_modules`, so Ithink installation is completed.

```json
    "@apexearth/egrep": {
      "version": "1.0.4",
      "resolved": "https://registry.npmjs.org/@apexearth/egrep/-/egrep-1.0.4.tgz",
      "integrity": "sha512-4NHcdVJnXGZoYxIh+ZieJYmzCBXs6BAVTuxtzMjdtqC+TMzdBAlmSMv92eX8Im6NNBRfjLduw0h3Mt+tA75M9Q==",
      "dev": true,
      "requires": {
        "commander": "^2.19.0",
        "glob": "^7.1.3",
        "recursive-readdir": "^2.2.2",
        "upath": "^1.1.0"
      }
    },
```

**Comment by apexearth** [comment]
Thanks for pointing out this issue!

I believe I've fixed it (version 1.0.5 on npm) and also updated the readme.

The example is now as follows:

```javascript
let egrep = require('@apexearth/egrep')
let stream = egrep({
    pattern: /test[1-9]/,
    files: [
        'test_files/file1.txt',
        'test_files/',
    ],
    recursive: true,
})
stream.on('data', data => {
    console.log(data)
})
stream.on('error', err => {
    console.log(err)
})
stream.on('close', () => {
    console.log('closed')
})
```

Note that we now `require('@apexearth/egrep')`

**Comment by exoego** [comment]
I can confirm that the import issue has been fixed.

Thanks for such quick fix 👍
